src/ui/components/SecondaryToolbox/react-devtools/utils/findProtocolObjectPropertyValue.ts

```diff
@@ -0,0 +1,10 @@
+import { Object as ProtocolObject } from "@replayio/protocol";
```

Comment:
:question: silly question, but why not have both of these in the same file?

Reply:
Your comment is below the `Object` type import but I think you're probably asking about the `findProtocolObjectProperty` utility function?

Just a subjective choice I guess. That function is called directly from other utility functions (like `findProtocolObjectPropertyValue`) so it makes more sense to me, organizationally, to have it in its own module. It's subjective though :)